.github/workflows/build_docs.yml

```diff
@@ -1,44 +1,48 @@
  name: Build Docs
- 
  on:
    push:
-     # Sequence of patterns matched against refs/tags
-     tags:
-       - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
-   workflow_dispatch:
-     inputs:
-       version:
-         description: Manual Doc Build Reason
-         default: test
-         required: false
+     branches:
+     - master
+     - main
  jobs:
    docs:
-     name: build & push docs
+     name: CI (${{ matrix.os }}-${{ matrix.environment-file }})
      runs-on: ${{ matrix.os }}
-     timeout-minutes: 90
+     continue-on-error: ${{ matrix.experimental }}
+     timeout-minutes: 20
      strategy:
        matrix:
          os: ['ubuntu-latest']
-         environment-file: [ci/310.yaml]
+         environment-file: [.ci/39.yml]
          experimental: [false]
      defaults:
        run:
          shell: bash -l {0}
-     
      steps:
-       - name: checkout repo
-         uses: actions/checkout@v4
-       
-       - name: setup micromamba
-         uses: mamba-org/provision-with-micromamba@main
+       - uses: actions/checkout@v2
+       - uses: actions/cache@v2
```

Comment:
What's the need for this `cache` action?

Reply:
not sure actually